docs/Remove-VSTeamAccessControlEntry.md

```diff
@@ -0,0 +1,141 @@
+
+
+
+# Remove-VSTeamAccessControlEntry
+
+## SYNOPSIS
+
+Removes specified ACEs in the ACL for the provided token. The request URI contains the namespace ID, the target token, and a single or list of descriptors that should be removed. Only supports removing AzD based users/groups.
+
+## SYNTAX
+
+## DESCRIPTION
+
+Removes specified ACEs in the ACL for the provided token. The request URI contains the namespace ID, the target token, and a single or list of descriptors that should be removed. Only supports removing AzD based users/groups.
+
+## EXAMPLES
+
```

Comment:
An example of using this would be great for those that have never seen it before.

Reply:
Added examples to help file in latest commit